http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/ConfigReaderSpec.scala

```diff
@@ -44,6 +44,12 @@ class ConfigReaderSpec extends AnyFlatSpec with Matchers {
           ),
           Set.empty
         )
+      ),
+      OidcAuthConfig(
+        Uri.unsafeFromString("https://fake"),
```

Comment:
This might need to be a real like `"https://accounts.google.com"` if you're actually instantiating a` OpenIDConnectConfiguration`. Or you could use `FakeOpenIDConnectConfiguration`

Reply:
This is probably why unit tests are failing